README.md

```diff
@@ -56,7 +56,9 @@ Note that when using glob patterns with `index.js`, you must enclose the pattern
 
 ### Configuration File
 
-Normally a config file is automatically generated (e.g. when you set a key using `index.js -k`) and updated (e.g. when you add a directory using `index.js -d`) but you can create your own config file `/etc/logdna.conf` on Linux and `C:\ProgramData\logdna\logdna.conf` on Windows and save your settings there:
+Normally a config file is automatically generated (e.g. when you set a key using `index.js -k`) and updated (e.g. when you add a directory
+using `index.js -d`) but you can create your own config file `/etc/logdna.conf` on Linux and `C:\ProgramData\logdna\logdna.conf` on Windows and
+save your settings there:
```

Comment:
Are you adding newlines so that it's all not on a single line?

Reply:
Are you arbitrarily cutting off lines based off of length or using a ruler- I know they have those in Sublime/VScode

Reply:
Yeah, this was all arbitrarily just cutting lines longer than 160 for readability.  We will eventually re-write all of this, but that shouldn't be until we have a solidified product (perhaps after, or right before, we split this repo).